opencti-platform/opencti-front/src/private/components/settings/Users.jsx

```diff
@@ -43,19 +43,24 @@ const Users = () => {
       },
       user_email: {
         label: 'Email',
-        width: '30%',
+        width: '25%',
         isSortable: true,
       },
       firstname: {
         label: 'Firstname',
-        width: '15%',
+        width: '12.5%',
         isSortable: true,
       },
       lastname: {
         label: 'Lastname',
-        width: '15%',
+        width: '12.5%',
         isSortable: true,
       },
+      effective_confidence_level: {
+        label: 'confidence',
+        width: '10%',
+        isSortable: false,
```

Comment:
effective confidence level is a computed value backend side, not an attribute in DB (depends on user's confidence + his groups confidence)

we cannot craft a query to do this computation, so we cannot paginate and sort properly this value.